package.json

```diff
@@ -9,12 +9,21 @@
   },
   "scripts": {
     "start": "node ./bin/probot run",
-    "test": "mocha && standard && npm run doc-lint",
+    "test": "jest && standard && npm run doc-lint",
     "doc-lint": "standard docs/**/*.md",
     "doc": "jsdoc -c .jsdoc.json",
     "postpublish": "script/publish-docs",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
+  "jest": {
+    "modulePathIgnorePatterns": [
+      "<rootDir>/__tests__/fixtures/",
```

Comment:
I'm not in love with the `__tests__` directory. The last jest project I saw still used `test`, and each test file ended in `.test.js`.  I liked that better.

Reply:
I'm +1 for changing the directory to `test`, but I'm not a big fan of `.test.js`. I find it repetitive, and you know that any file within `/test/` is a test. I'm not _that_ opposed to it, so tell me if I should ignore my inner voice.

Reply:
The most compelling reason is that it's easier to switch between tabs in your editor.  If you have `lib/index.js` and `test/index.js` open, some editors make it difficult to find which file you want to jump to.

I don't really care either way, so go with whatever you prefer.

Reply:
I would also cast my vote for a `test` directory. I don't mind the extra repetition so much.